lib/ruby_indexer/lib/ruby_indexer/visitor.rb

```diff
@@ -46,6 +46,18 @@ def visit(node)
 
         name = fully_qualify_name(node.target.location.slice)
         add_constant(node, name)
+      when YARP::MultiWriteNode
```

Comment:
I guess we need this gymnastic of accessing the node targets manually rather than visiting down the tree just to access the value?

I wonder if our simplified visitor could descend into two ConstantWriteNode to make it easier to handle for free?

Reply:
Yes, it's because we need the value. That would simplify the code a bit indeed, but we'd need to be careful about missing values.

E.g.:
```ruby
# In this case, the first ConstantWrite visit would have value C and the second nil
A, B = C
```